index.html

```diff
@@ -762,9 +762,311 @@
         <!-- ************ -->
 
         <section id="class-3">
-          <h2>Third Class: Jeff</h2>
+          <h2>Third Class</h2>
+          <p>Jeff O'Leary</p>
         </section>
+        <section>
+          <h4>Agenda</h4>
+          <p class="fragment">Navigation</p>
```

Comment:
Ahhh yes yes, I'll make a slide like "what we've learned previously and how that fits into today's class".  Good call man, I definitely planned on referring back to what was previously learned and how it relates to everything I'm teaching.  For example, explaining how pushViewController is a function called on an instance of UINavigationController class.